mmcv/runner/runner.py

```diff
@@ -388,7 +388,10 @@ def run(self, data_loaders, workflow, max_epochs, **kwargs):
     def register_lr_hook(self, lr_config):
         if isinstance(lr_config, dict):
             assert 'policy' in lr_config
-            hook_type = lr_config.pop('policy').title() + 'LrUpdaterHook'
+            policy_type = lr_config.pop('policy')
+            if policy_type == policy_type.lower():
+                policy_type = policy_type.title()
+            hook_type = policy_type + 'LrUpdaterHook'
```

Comment:
Add some comments for the above logic, otherwise users may get confused.

Reply:
Sure